README.md

```diff
@@ -31,34 +31,34 @@ Test Driven Development (TDD) makes your team a well-oiled machine which means y
 Once you have a ***suite*** of tests that run on every change, you will
 begin to develop a whole other level of ***confidence*** in your codebase
 and will discover a new freedom to be ***creative*** without fear of
-"*breaking*" anything else; truly *game-changing*.
+"*breaking*" anything unexpectedly; truly *game-changing*.
 
 
 ## What?
 
 This tutorial will help you get started with
 **T**est **D**riven **D**evelopment (**TDD**) *today*! <br />
 In the next ***30 minutes*** you will learn _everything_<sup>1</sup>
-you need to know to write tests for your web project!
+you need to write tests for your web project!
```

Comment:
the "know" was included to emphasise the removal of the knowledge barrier. 🤔 
but I agree that the sentence can be simplified. 👍